enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/configuration/IdentityVerificationConfig.java

```diff
@@ -51,11 +53,11 @@ public class IdentityVerificationConfig {
     @Value("${enrollment-server-onboarding.presence-check.cleanupEnabled:false}")
     private boolean presenceCheckCleanupEnabled;
 
-    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:1}")
-    private int dataRetentionTime;
+    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:PT1H}")
```

Comment:
I guess we can use also the basic format as `1h` here

https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.conversion.durations

Reply:
Sure, we can, but I will update it in all occurrences.